### ISSUE TITLE
Not compatible with Mac silicon

### ISSUE THREAD

**Issue by arnaudgout** [aws/aws-gamekit-unity#9]
Spent a few hours trying to have the libraries properly loaded because I had several of this message after having installed the package:

> DllNotFoundException: aws-gamekit-core assembly:<unknown assembly> type:<unknown type> member:(null)
AWS.GameKit.Runtime.Core.CoreWrapper+<>c__DisplayClass93_1.<DeploymentOrchestratorInstanceCreate>b__0 () (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Runtime/Scripts/Core/CoreWrapper.cs:401)
AWS.GameKit.Runtime.Utils.DllLoader.TryDll[T] (System.Func`1[TResult] importedMethod, System.String dllName, T returnOnError) (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Runtime/Scripts/Utils/DllLoader.cs:41)
UnityEngine.Debug:LogException(Exception)
AWS.GameKit.Runtime.Utils.<>c__DisplayClass40_0:<LogException>b__0() (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Runtime/Scripts/Utils/Logging.cs:127)
AWS.GameKit.Runtime.Utils.Logging:LogInUnity(Level, Action) (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Runtime/Scripts/Utils/Logging.cs:211)
AWS.GameKit.Runtime.Utils.Logging:LogException(Object, Exception) (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Runtime/Scripts/Utils/Logging.cs:125)
AWS.GameKit.Runtime.Utils.DllLoader:LogDllException(Exception, String) (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Runtime/Scripts/Utils/DllLoader.cs:99)
AWS.GameKit.Runtime.Utils.DllLoader:TryDll(Func`1, String, IntPtr) (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Runtime/Scripts/Utils/DllLoader.cs:45)
AWS.GameKit.Runtime.Core.CoreWrapper:DeploymentOrchestratorInstanceCreate(String, String) (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Runtime/Scripts/Core/CoreWrapper.cs:401)
AWS.GameKit.Editor.Core.FeatureDeploymentOrchestrator:.ctor(ICoreWrapperProvider, String, String) (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Editor/Scripts/Core/FeatureDeploymentOrchestrator.cs:35)
AWS.GameKit.Editor.Core.FeatureDeploymentOrchestrator:.ctor(String, String) (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Editor/Scripts/Core/FeatureDeploymentOrchestrator.cs:42)
AWS.GameKit.Editor.GameKitEditorManager:.ctor() (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Editor/Scripts/GameKitEditorManager.cs:76)
System.Lazy`1:get_Value()
AWS.GameKit.Common.Singleton`1:Get() (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Common/Scripts/Utils/Singleton.cs:22)
AWS.GameKit.Editor.GameKitEditorManager:.cctor() (at Library/PackageCache/com.amazonaws.gamekit@4e6d95d6d0fc/Editor/Scripts/GameKitEditorManager.cs:57)
UnityEditor.EditorAssemblies:ProcessInitializeOnLoadAttributes(Type[]) (at /Users/bokken/build/output/unity/unity/Editor/Mono/EditorAssemblies.cs:126)

Long story short, after having dug a bit and modified the cache version of the library, I managed to get to the actual interesting message: not supported architecture, those dylib are for x86, not arm64.
